beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -107,18 +126,32 @@ func (s *Service) roundRobinSync(genesis time.Time) error {
 		}).Debug("Sending batch block request")
 		resp, err := blocksFetcher.requestBlocks(ctx, req, best)
 		if err != nil {
-			log.WithError(err).Error("Failed to receive blocks, exiting init sync")
-			return nil
+			log.WithError(err).Error("Failed to receive blocks")
+			nxtPeer := nextBestPeer(best)
+			if nxtPeer != "" {
+				best = nxtPeer
+			}
+			continue
```

Comment:
1. If `nxtPeer == ""`, shouldn't we at least warn user on start of new iteration -- we may even wait for min-peers to sync. Otherwise, if list of peers is empty (which is possible, I suppose?), we will spin very quickly w/o requesting anything.
1. If you don't have strong opinion about it, can you please name var as `nextPeer` instead of `nxtPeer` - it will be consistent to `nextBestPeer()` and overall read easier (w/o becoming too large of a name).